lib/osml/tile_server/ts_dataplane.ts

```diff
@@ -222,6 +224,22 @@ export class TSDataplane extends Construct {
       }
     });
 
+    // Create a Lambda function to clean up the TS DLQ
```

Comment:
How does this get triggered? I might have an optional flag to enable the integ test and create it by default with a function rather than make it in here... but we can make an action to go back and clean that up later.

Reply:
It get triggered once there's a message in `TSQueueDLQ` queue. If you go further down to Line 243

```
// Attach DLQ Queue to Lambda Sweeper Function
this.sqsDlqEventSource = new SqsEventSource(this.jobQueue.dlQueue);
this.lambdaSweeperFunction.addEventSource(this.sqsDlqEventSource);
```